Test/Web/FunctionalTests/FunctionalTests/Helpers/EtwEventSession.cs

```diff
@@ -94,7 +95,11 @@ private bool IsInterestingTrace(TraceEvent data)
 
             bool result = true;
 
-            if (data.PayloadNames.Length > 0)
+            if (data.ProviderName == "Microsoft-AspNet-Diagnostics")
+            {
+                result = true;
```

Comment:
>result = true; [](start = 16, length = 14)

we check later on that the trace is coming from the process we monitor. Simple `return true` can mess up traces from the parrallel tests executions.